tutorials/animation/animation_track_types.rst

```diff
@@ -57,6 +57,80 @@ inspector dock. There, you can change the method to call. If you expand the
 
 .. image:: img/node_method_args.webp
 
+To create such a track through code, pass a dictionary that contains the target method's name
+and parameters as the Variant for ``key`` in ``Animation.track_insert_key()``. The keys and
+their expected values are as follows:
+
++---------------------------------------+-----------------------------------------------------------------------------+
+| **Key**                               | **Value**                                                                   |
++=======================================+=============================================================================+
+| ``"method"``                          | The name of the method as a ``String``                                      |
++---------------------------------------+-----------------------------------------------------------------------------+
+| ``"args"``                            | The arguments to pass to the function as an ``Array``                       |
++---------------------------------------+-----------------------------------------------------------------------------+
+
+.. tabs::
+ .. code-tab:: gdscript GDScript
+
+    # Create a call method track
+    func create_method_animation_track():
+        # Get or create the animation the target method will be called from
+        var animation = $AnimationPlayer.get_animation("idle")
+        # Get or create the target method's animation track
+        var track_index = animation.add_track(Animation.TYPE_METHOD)
+        # Make the arguments for the target method jump()
+        var jump_velocity = -400.0
+        var multiplier = randf_range(.8, 1.2)
+        # Get or create a dictionary with the target method's name and arguments
+        var method_dictionary = {
+            "method": "jump",
+            "args": [jump_velocity, multiplier],
+        }
+
+        # Set scene-tree path to node with target method
+        animation.track_set_path(track_index, ".")
+        # Add the dictionary as the animation method track's key
+        animation.track_insert_key(track_index, 0.6, method_dictionary, 0)
+
+
+    # The target method that will be called from the animation
+    func jump(jump_velocity, multiplier):
+        velocity.y = jump_velocity * multiplier
+
+ .. code-tab:: csharp
+
+    // Create a call method track
+    public void CreateAnimationTrack()
+    {
+        // Get reference to the AnimationPlayer
+        var animationPlayer = GetNode("AnimationPlayer") as AnimationPlayer;
```

Comment:
We tend to prefer using the generic version of this method in the documentation.

```suggestion
        var animationPlayer = GetNode<AnimationPlayer>("AnimationPlayer");
```